File: Num2e2.py
import psycopg2
def  post(x,y,z):
    try:
        conexion = psycopg2.connect(
            host = "localhost",
            port = "5432",
            user = "postgres",
            password = "123fgthg",
            dbname = "HWP"
            )
    except:
        print("Sin Conexion Exitosa\n")
            
    cursor = conexion.cursor()
    cursor.execute("INSERT INTO plus2(numstr, numend, num2num) VALUES (%s, %s, %s);", (x, y,z))
    conexion.commit()
    cursor.close()
    conexion.close()
print("Numeros de dos en dos\n")
st=int(input("Ingrese el numero de inicio:\n"))
en=int(input("Ingrese el numero de fin:\n"))
r=[]
for x in range(st,en+1,2):
    r.append(x)

print("Los numeros son:", str(r))
post(st, en, r)
exit(0)

File: Num2e2.m
clc; clear all; close all;
st=input('Ingrese el numero de inicio:  ');
en=input('Ingrese el numero de fin:  ');
cont=1;
for x=st:2:en
  V(cont)= x;
  cont=cont+1;
endfor
disp(V)
params= cell(1,3);
params{1,1}=st;
params{1,2}=en;
s=mat2str(V);
params{1,3}=s;
save('Num2e2.txt','V','-ascii');
conn = pq_connect(setdbopts('dbname','HWP','host','localhost',
'port','5432','user','postgres','password','123fgthg'));
N=pq_exec_params(conn, "insert into plus2(numStr, numEnd, num2num) values($1,$2,$3);",params); %insertar datos en la tabla
